modules/example/resources/smithy4s.example.ObjectService.json

```diff
@@ -140,8 +140,7 @@
                         "properties": {
                             "int": {
                                 "type": "number",
-                                "format": "int32",
-                                "nullable": true
+                                "format": "int32"
```

Comment:
Looks like EnumShapes are maybe rendered differently than the old EnumTraits were by our openapi conversion. Going to need to take a look.

Reply:
Does this actually have something to do with the changes to get rid of boxed types in smithy?

Reply:
It might be the case indeed